templates/essentials.yaml

```diff
@@ -402,6 +402,17 @@ Resources:
               - "ec2:Describe*"
               - "ec2:CreateTags"
             Resource: "*"
+          - Sid: DenyWriteProtectedPath
```

Comment:
changing this has a cascading effect of updating TagRootVolumeProfile and TagRootVolumeRole.  My concern is that this may cause any instances using TagRootVolumeProfile and TagRootVolumeRole directly to re-provision.  Have you tested that?

Reply:
I have tested whether an instance rebuilds after a change to a policy referenced in its instance profile, and it does not force a rebuild.

Reply:
sound good